index.html

```diff
@@ -5,9 +5,51 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/b89b7c8adf.js" crossorigin="anonymous"></script>
 </head>
 <body>
+
+  <main>
+
+    <nav>
+      <div class="logo"><h2>AS Pathak</h2></div> 
+      <img class="menu" src="media\Union.png" alt="hamburger" />
+    </nav>
+
+    <div class="introduction">
```

Comment:
I suggest you change this from a `<div>` tag to a `<section>` tag. It represents a generic standalone section of the document [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.I